src/components/layout/AppBar.vue

```diff
@@ -199,6 +226,19 @@ export default class AppBar extends Mixins(StateMixin) {
     })
   }
 
+  handleHostShutdown () {
+    this.$confirm(
+      this.$tc('app.general.simple_form.msg.confirm_shutdown_host'),
+      { title: this.$tc('app.general.label.confirm'), color: 'card-heading', icon: '$error' }
+    )
+      .then(res => {
+        if (res) {
+          this.$emit('click')
+          this.hostShutdown()
+        }
+      })
+  }
```

Comment:
```suggestion
  async handleHostShutdown () {
    const res = await this.$confirm(
      this.$tc('app.general.simple_form.msg.confirm_shutdown_host'),
      { title: this.$tc('app.general.label.confirm'), color: 'card-heading', icon: '$error' }
    )

    if (res) {
      this.hostShutdown()
    }
  }
```

That `$emit` call should not be needed here (that was on the `SystemCommands.vue` as it probably was needed to dismiss the popup menu, but that is not the case here!

Ideally, I think we should move from `Promise` based code to more modern `async/await` calls, so I went ahead and refactored this method!